docs/references/subcommands/analyze.md

```diff
@@ -67,6 +67,14 @@ The Vendored Dependencies feature allows you to scan for licenses directly in yo
 | `--force-vendored-dependency-scan-method` | Force the vendored dependency scan method. The options are 'CLILicenseScan' or 'ArchiveUpload'. 'CLILicenseScan' is usually the default unless your organization has overridden this. |
 | `--force-vendored-dependency-rescans`     | Force vendored dependencies to be rescanned even if the revision has been previously analyzed by FOSSA. This currently only works for CLI-side license scans.                         |
 
+### Custom License Searches
+
+The "Custom License Searches" feature allows you to define regular expressions that are used to search through your codebase. If a match to the regular expression is found, then a custom license is reported for that match.
```

Comment:
```suggestion
The "Custom License Searches" feature allows you to define text strings or regular expressions that are used to search through your codebase. If a match to the text string or regular expression is found, then a custom license is reported for that match.
```
This is more wordy, but maybe this is useful to be less intimidating for users not familiar with regexes who may not even need all that functionality?